dev/CommandBarFlyout/CommandBarFlyout.xaml

```diff
@@ -376,6 +377,8 @@
                                         contract4Present:Control.IsTemplateKeyTipTarget="True"
                                         IsAccessKeyScope="True"
                                         IsTabStop="False"
+                                        contract7Present:CornerRadius="{TemplateBinding CornerRadius}"
+                                        contract7NotPresent:CornerRadius="{ThemeResource OverlayCornerRadius}"
```

Comment:
I think this is the line causing your build issues.  CornerRadius wasn't added on Controls until RS5 (it was added to panels in TH1). So you can't set the CornerRadius property on this button when contract 7 isn't present.